tidal_per_transformers/transformers/artist_filter_transformer.py

```diff
@@ -48,22 +48,27 @@ def apply_filters(category_filters: DataFrame,
                       streamers_count: int,
                       drop_holiday: bool,
                       drop_ambient: bool,
-                      drop_children: bool) -> DataFrame:
+                      drop_children: bool,
+                      artist_column_for_filter: str) -> DataFrame:
         """Applies filters given in a data frame
 
-        :param category_filters:    dataframe containing filters
-        :param stream_count:        min stream count
-        :param streamers_count:     min streamers count
-        :param drop_holiday:        flag to drop holiday
-        :param drop_ambient:        flag to drop ambient music
-        :param drop_children:       flag to drop children
-        :return: cleaned dataframe
+        :param category_filters:            dataframe containing filters
+        :param stream_count:                min stream count
+        :param streamers_count:             min streamers count
+        :param drop_holiday:                flag to drop holiday
+        :param drop_ambient:                flag to drop ambient music
+        :param drop_children:               flag to drop children
+        :param artist_column_for_filter:    column containing artists
+        :return:                            cleaned dataframe
         """
-        return apply_category_filters(dataframe=category_filters
-                                      .where(F.col(c.AVAILABLE))
-                                      .where(F.col(c.NON_MUSIC) == 0)
-                                      .where(F.col(c.STREAM_COUNT) >= stream_count)
-                                      .where(F.col(c.STREAMERS_COUNT) >= streamers_count),
-                                      drop_holiday=drop_holiday,
-                                      drop_ambient=drop_ambient,
-                                      drop_children=drop_children).select(c.ARTIST_ID)
+        all_checks = category_filters.where((F.col(c.AVAILABLE) == False) |
```

Comment:
Which job updates the available col in content filter table?
It is false for some new releases this morning.

Reply:
I think a lot of jobs use this, we can search in the internal repos.
what I did is I basically flipped all the conditions, and did a left anti instead of an inner join, this will filter our all tracks meeting each criteria by default, but leave the ones that are not covered by our filters jobs